sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### New Features
 - Added properties `Value` and `ExpectedValue` to `DataPointAnomaly` to provide more information about the anomalous data point.
 - Added properties `ValueOfRootNode` and `ExpectedValueOfRootNode` to `AnomalyIncident` to provide more information about the anomalous data point at the root node of the indicent.
+- Headers that were marked as `REDACTED` in error messages and logs are now exposed by default.
```

Comment:
do you think it is worth saying `Request/Response` headers ....

Reply:
Updated it to "Response headers" since the change doesn't affect headers in the request.